src/main/java/model/LottoTicket.java

```diff
@@ -0,0 +1,60 @@
+package model;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    public static final int LOTTO_TICKET_PRICE = 1000;
+    private final int LOTTO_SIZE = 6;
+
+    private final List<LottoNumber> numbers;
+
+    public LottoTicket(List<LottoNumber> numbers) {
+        validateNumbers(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validateNumbers(List<LottoNumber> numbers) {
+        if (numbers.size() != LOTTO_SIZE){
+            throw new IllegalArgumentException("로또 티켓은 6개의 번호로 구성되어야 합니다.");
```

Comment:
[Comment]

에러 문구도 코드의 관리 대상이죠.
매직넘버를 제거한 김에 재활용해보는 것은 어떤가요?
```suggestion
             throw new IllegalArgumentException("로또 티켓은 " + LOTTO_SIZE + "개의 번호로 구성되어야 합니다.");
```

Reply:
에러 문구까지는 생각을 못했네요! 좋은 제안 감사합니다:)